src/filesystem.lisp

```diff
@@ -73,3 +73,15 @@
     wallet))
 
 
+(defun new-temporary-directory (&key (root #p"/var/tmp/emotiq/"))
+  (loop
+     :with sub-directory = (make-pathname
```

Comment:
@dbmcclain Should be 
```
  :for sub-directory = (make-pathname
``` fixed in `dev` with <https://github.com/emotiq/emotiq/commit/ea35ebe2feab2edbf361c1e3438efc09b1726433>

Reply:
If you say so... I didn't write this, but if you like, I will edit here online